src/core/reference/include/openvino/reference/select.hpp

```diff
@@ -11,6 +11,13 @@
 
 namespace ov {
 namespace reference {
+namespace func {
+template <class T>
+T select(char s, T x, T y) {
```

Comment:
As in other similar reference add comment why used instead of lambda (bin-size reduction?)

Reply:
It appeared it doesn't bring binary size reduction, though this changed has been reverted.